tests/messages/test_frontend.py

```diff
@@ -1535,6 +1535,12 @@ def test_extract_cli_knows_dash_s():
     assert cmdinst.strip_comments
 
 
+def test_extract_cli_knows_dash_dash_last_dash_translator():
+    cmdinst = configure_cli_command("extract --last-translator e-mail -o foo babel")
+    assert isinstance(cmdinst, ExtractMessages)
+    assert cmdinst.last_translator
```

Comment:
This should presumably check that it was assigned to `e-mail`?

Reply:
Sure you are right. Actually it can be not only an e-mail, fixed.